### PR TITLE
Update layer layout and carousel storybook examples, remove border from layer and add animations to layer layout

### PR DESCRIPTION
**Pull request by JoanaMMoreira**
- Rename layer examples
- Remove border from layer component
- Add fade animations to scrim
- Update carousel storybook example
- Update existing layer storybook examples
- Add new layer real life examples for centre, left, top, right and bottom positions

### CODE REVIEW

File: .changeset/plenty-cows-train.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+"@jpmorganchase/uitk-lab": minor
+---
+
+Rename layer examples
```
Comment (ZarkoPernar) [inline .changeset/plenty-cows-train.md:5]: Do you think non-user facing changes should be added to the change log?

Comment (JoanaMMoreira) [reply .changeset/plenty-cows-train.md]: That's a good point, I don't think they should so I've updated the change log

Comment (ZarkoPernar) [reply .changeset/plenty-cows-train.md]: Although, I've seen some libraries add documentation edits in to the change log

Comment (JoanaMMoreira) [reply .changeset/plenty-cows-train.md]: @origami-z @joshwooding what do you reckon?

Comment (joshwooding) [reply .changeset/plenty-cows-train.md]: Hmm, I think for now leaving them out is fine. Potentially in the future we can add them.

File: .changeset/plenty-cows-train.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+---
+"@jpmorganchase/uitk-lab": minor
+---
+
+Remove border from layer component
+Add fade animations to scrim
```
Comment (joshwooding) [inline .changeset/plenty-cows-train.md:6]: Scrim doesn't actually have animations, the LayerLayout does.

**Review by joshwooding — CHANGES_REQUESTED** [review]
See comment

**Review by joshwooding — APPROVED** [review]

**Pull request merged**